pkgs/applications/misc/dotfiles/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchurl, python27Packages }:
+
+python27Packages.buildPythonPackage rec {
+  name = "dotfiles-${version}";
+  version = "0.6.4";
+
+  meta = with stdenv.lib; {
+    description = "Easily manage your dotfiles";
+    homepage = https://github.com/jbernard/dotfiles;
+    license = licenses.isc;
+  };
+
+  src = fetchurl {
+    # https://github.com/jbernard/dotfiles/archive/v${version}.tar.gz
+    url = "mirror://pypi/d/dotfiles/${name}.tar.gz";
+    sha256 = "03qis6m9r2qh00sqbgwsm883s4bj1ibwpgk86yh4l235mdw8jywv";
+  };
+
+  buildInputs = with python27Packages; [ pytest ];
```

Comment:
`checkInputs` (note that his only a Python thing)

Reply:
done.